CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.4.5
+
+* Fix some issues on xenial.
```

Comment:
Can you make this a little more specific, please?

Reply:
Sure :)